kubeless-rbac-novols.jsonnet

```diff
@@ -0,0 +1,9 @@
+# Remove volumeClaimTemplates from kafkaSts to enable testing kubeless
```

Comment:
oh, I just wanted to minimise the changes to the service for test vs prod (and we might have more pv-using things in future so it would avoid having to make more -novols versions).  I also appreciate not wanting to spend further time on this PR, however :P